src/storage/src/metrics/source.rs

```diff
@@ -57,11 +56,6 @@ impl GeneralSourceMetricDefs {
         Self {
             // TODO(guswynn): some of these metrics are not clear when subsources are involved, and
             // should be fixed
-            capability: registry.register(metric!(
```

Comment:
I removed this metric as it doesn't seem to be all that useful. We don't even have a panel for it in grafana and we have a separate one that tracks the frontier of each subsource. Since this metric was populated from an operator I deleted I chose to remove it as well instead of finding a new home for it